### PR TITLE
Added Kimai favicon

### PR DESCRIPTION
**Pull request by joejoe31b**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -82,6 +82,7 @@
 		}
 	});
 	</script>
+	<link href="favicon.ico" rel="shortcut icon"></link>
```
Comment (simonschaufi) [inline index.php]: this line is not needed as favicon.ico files in root directory are always loaded.

Comment (joejoe31b) [inline index.php]: Are you sure? I needed that line for it to display for me with Firefox 47.0.1 and Chromium 52 (on Arch Linux). Does it display for you without that line?

Comment (kevinpapst) [inline index.php]: The main problem is, that this "app" does not ship a favcon and your line only works in smoe circuminstances. If somebody has Kimai in a subfolder, there is probably no favicon.ico at that URL.

Comment (joejoe31b) [inline index.php]: My commit adds the Kimai `favicon.ico` file to the root directory of kimai-mobile. @simonschaufi was saying how there is no need to link the favicon in kimai-mobile's `index.php` as the browser should use the favicon from the same directory as the index automatically.

This didn't work for me, and I needed to add the line I added in `index.php` for it to work for me.

Comment (kevinpapst) [inline index.php]: Ah, I didn't see that. Right. Thanks!
favicon.ico is only requested for the root URL of the domain IMO - so your commit "fixes" possible 404 as well.

**Comment by kevinpapst** [comment]
Thanks @joejoe31b

**Pull request merged**